app/templates/clarifications.html

```diff
@@ -211,19 +207,21 @@ <h6>Unless expressly stated, the following clarifications apply:</h6>
       <h5 class="--padding">Edit clarification</h5>
```

Comment:
Do we need specific class for padding? It will be impossible to support it later on